contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -124,8 +272,55 @@ NXmpes(NXobject):
             doc: |
               Description of the detector type.
             enumeration: [DLD, Phosphor+CCD, Phosphor+CMOS, ECMOS, Anode, Multi-anode]
-          (NXdata):
+          device_information(NXfabrication):
```

Comment:
What is the default you demand if any (so that H5Web has something to pick up?)

Reply:
I dont' understand what you mean by default here. It is a recommended field, so there's no default needed. Could you please clarify?

Reply:
Yeah that was a poor comment, my thought at that time back then was "do you still assure you will have at least one NXdata that H5Web can pick up from or not?"

Reply:
Since `data(NXdata)` is now required, we always have this.